### PR TITLE
CLOUDP-108862: Get a specific GCP Private Endpoint for your project

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes

_Jira ticket:_ [CLOUDP-108862](https://jira.mongodb.org/browse/CLOUDP-108862)

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
➜  mongocli git:(CLOUDP-108862) ✗ mongocli atlas privateEndpoint gcp describe 61e055436d861a60d7ea7664
ID                         REGION       STATUS      ERROR
61e055436d861a60d7ea7664   CENTRAL_US   AVAILABLE
```

- Note: e2e tests will be merged after create command is introduced as there is a dependency between e2e tests.

### CODE REVIEW

File: internal/cli/atlas/privateendpoints/gcp/describe.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+func DescribeBuilder() *cobra.Command {
+	opts := new(DescribeOpts)
+	cmd := &cobra.Command{
+		Use:     "describe <ID>",
+		Aliases: []string{"get"},
+		Args:    require.ExactArgs(1),
+		Short:   "Return a specific GCP private endpoint for your project.",
+		Annotations: map[string]string{
+			"args":         "ID",
+			"requiredArgs": "ID",
```
Comment (andreaangiolillo) [inline internal/cli/atlas/privateendpoints/gcp/describe.go:68]:
```suggestion
			"args":         "privateEndpointId",
			"requiredArgs": "privateEndpointId",
```

File: internal/cli/atlas/privateendpoints/gcp/describe.go [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas privateEndpoint(s)|privateendpoint(s) gcp describe|get <ID> [--projectId projectId].
+func DescribeBuilder() *cobra.Command {
+	opts := new(DescribeOpts)
+	cmd := &cobra.Command{
+		Use:     "describe <ID>",
```
Comment (andreaangiolillo) [inline internal/cli/atlas/privateendpoints/gcp/describe.go:62]:
```suggestion
		Use:     "describe <privateEndpointId>",
```

File: internal/cli/atlas/privateendpoints/gcp/describe.go [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+	r, err := opts.store.PrivateEndpoint(opts.ConfigProjectID(), provider, opts.id)
+
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas privateEndpoint(s)|privateendpoint(s) gcp describe|get <ID> [--projectId projectId].
```
Comment (andreaangiolillo) [inline internal/cli/atlas/privateendpoints/gcp/describe.go:58]:
```suggestion
// mongocli atlas privateEndpoint(s)|privateendpoint(s) gcp describe|get <privateEndpointId> [--projectId projectId].
```

File: internal/cli/atlas/privateendpoints/gcp/gcp.go [hunk 4]
```diff
@@ -21,11 +21,15 @@ import (
 const provider = "GCP"
 
 func Builder() *cobra.Command {
+	const use = "gcp"
 	cmd := &cobra.Command{
-		Use:   "gcp",
+		Use:   use,
```
Comment (andreaangiolillo) [inline internal/cli/atlas/privateendpoints/gcp/gcp.go:26]:
```suggestion
		Use:  "gcp",
```

File: internal/cli/atlas/privateendpoints/gcp/describe.go [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+	opts := new(DescribeOpts)
+	cmd := &cobra.Command{
+		Use:     "describe <ID>",
+		Aliases: []string{"get"},
+		Args:    require.ExactArgs(1),
+		Short:   "Return a specific GCP private endpoint for your project.",
+		Annotations: map[string]string{
+			"args":         "ID",
+			"requiredArgs": "ID",
+			"IDDesc":       "Unique identifier of the private endpoint service that you want to retrieve.",
```
Comment (andreaangiolillo) [inline internal/cli/atlas/privateendpoints/gcp/describe.go:69]:
```suggestion
			"privateEndpointIdDesc":       "Unique identifier of the private endpoint service that you want to retrieve.",
```

**Review by andreaangiolillo — COMMENTED** [review]
Great work! 💯  just a few small changes

File: internal/cli/atlas/privateendpoints/gcp/describe.go [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+	opts := new(DescribeOpts)
+	cmd := &cobra.Command{
+		Use:     "describe <privateEndpointId>",
+		Aliases: []string{"get"},
+		Args:    require.ExactArgs(1),
+		Short:   "Return a specific GCP private endpoint for your project.",
+		Annotations: map[string]string{
+			"args":                  "privateEndpointId",
+			"requiredArgs":          "privateEndpointId",
+			"privateEndpointIdDesc": "Unique identifier of the private endpoint service that you want to retrieve.",
```
Comment (JuliaMongo) [inline internal/cli/atlas/privateendpoints/gcp/describe.go:69]: Please see the other PRs -- they had a different description for this, with "22 character..." stuff. Use that description. Also please add an Example: key, after Annotations:, with an example of running this command.

**Review by JuliaMongo — CHANGES_REQUESTED** [review]
Need to make the description for the ID match other similar descriptions in recent PRs.
Add an Example: annotation please.

**Review by gssbzn — APPROVED** [review]
LGTM

File: internal/cli/atlas/privateendpoints/gcp/describe.go (hunk 6 above)
Comment (blva) [reply internal/cli/atlas/privateendpoints/gcp/describe.go:69]: Thanks Julia, will update!

**Review by JuliaMongo — APPROVED** [review]

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**